src/pages/Layouts/Member/index.js

```diff
@@ -0,0 +1,143 @@
+        )
+      )
+      setCurrentPage(1)
+    }
+  }, [selectedPosition, selectedGeneration])
+
+  const handlePageChange = (page) => {
+    setCurrentPage(page);
+  }
+
+  const totalPages = Math.ceil(filteredMember.length / profilesPage)
+  const startIndex = (currentPage - 1) * profilesPage
+  const endIndex = startIndex + profilesPage
+
+  return (
+    <S.Layout>
+      <S.Titles>
+        <S.MainTitle>TEAM MEMBER</S.MainTitle>
+        <S.SubTitle>끊임없이 나아가는 사람들</S.SubTitle>
+      </S.Titles>
+      <S.GenerationButtons>
+        {
+          generations.map((item, index) => {
+            return (
+              <S.Generation
+                onClick={() => {
+                  let newGeneration;
+                  if (selectedGeneration.includes(item)) {
+                    newGeneration = selectedGeneration.filter((activeGen) => item !== activeGen)
+                  } else {
+                    newGeneration = [...selectedGeneration, item]
+                  }
+                  setSelectedGeneration(newGeneration)
+                }
+                }
+                selected={selectedGeneration.includes(item)}
+              >
+                {item}기
+              </S.Generation>
+            )
+          })
+        }
+      </S.GenerationButtons>
+      <S.PositionButtons>
+        {
+          positions.map((item, index) => {
+            return (
+              <S.Position
+                onClick={() => {
+                  let newPosition;
+                  if (selectedPosition.includes(item)) {
+                    newPosition = selectedPosition.filter((activePos) => item !== activePos)
+                  } else {
+                    newPosition = [...selectedPosition, item]
+                  }
+                  setSelectedPosition(newPosition)
+                }
+                }
+                selected={selectedPosition.includes(item)}
+              >
+                {item}
+              </S.Position>
+            )
+          })
+        }
+      </S.PositionButtons>
+      <S.MemberLayout>
+        <S.LArrowIconButton
+          onClick={() => handlePageChange(currentPage - 1)}
+          disabled={currentPage === 1}
+        >
+          <S.ArrowIconImg src={Arrow} style={{ transform: 'scaleX(-1)' }} />
+        </S.LArrowIconButton>
+        <S.MembersBox>
+          {
+            filteredMember?.slice(startIndex, endIndex).map((item, index) => {
+              return (
+                <S.CoverLink href={`https://github.com/${item.githubId}`} target='_blank' key={index}>
+                  <S.Members>
+                    <S.MemberImg src={item.profile_url} />
+                    <S.Content>
+                      <S.Name>
+                        {item.name ? item.name : item.githubId}
+                      </S.Name>
+                      <S.PosAndGen>{item.position} | {item.generation}기</S.PosAndGen>
+                    </S.Content>
+                    <S.GithubImg src={Github} />
```

Comment:
```suggestion
                    <S.GithubImg src={githubImage} />
```
다음과 같이 네이밍을 좀 더 명시적으로 지어주면 좋을것같아여

Reply:
변경햇습니다!